requirements.txt

```diff
@@ -1,15 +1,21 @@
-# for all
 numpy>=1.24.2
 wandb>=0.13.10
 matplotlib>=3.7.0
 scipy>=1.10.0
 pytorch-lightning>=2.0.3
 shapely>=2.0.1
 networkx>=3.0
-Cartopy>=0.22.0
+Cartopy>=0.20.0
 pyproj>=3.4.1
 tueplots>=0.0.8
 plotly>=5.15.0
+torch>=2.0.1
+--find-links https://data.pyg.org/whl/torch-${TORCH}.html
```

Comment:
I think you are right Leif, this doesn't work on CPU only on GPU. I must have gotten confused, because I always install pytorch with conda beforehand. Short answer, my solution sucks and we should find a working solution for the soon-to-be packaged repo. Did you already find a way to install torch that is both flexible and robust enough across various machines?

Reply:
Ah I see :) ok, thanks for the clarification. I will think about this a bit more and give a proposition